docsource/development.rst

```diff
@@ -52,7 +52,7 @@ Learn from existing migration scrips
 
 
 Since version 14, the migration scripts are located in:
-``OpenUpgrade/openupgrade-framework/openupgrade_scripts/scripts/<module-name>``
+``OpenUpgrade/openupgrade_scripts/scripts/<module-name>``
```

Comment:
It probably makes sense to drop the `OpenUpgrade` part here, analogous to the mention above.